filechecker.py

```diff
@@ -0,0 +1,17 @@
+import os
+import argparse
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', type=checkpath)
+    return parser.parse_args()
+
+def checkpath(path):
+    if os.path.exists(path):
+        print("Valid path!")
+        return path
+    else:
+        raise argparse.ArgumentTypeError(f"Not a valid path")
+
+
+parsed_args = parse_arguments()
```

Comment:
In Python it is a practice to use `if __name__ == '__main__':` to enclose definitions of executables (that is what you are doing). Checkout: https://stackoverflow.com/questions/22492162/understanding-the-main-method-of-python on what it means and how to do it.